packages/devextreme-scss/scss/widgets/fluent/chat/_sizes.scss

```diff
@@ -0,0 +1,49 @@
+@use "../sizes" as *;
+
+// adduse
+
+$chat-bubble-border-radius: null !default;
+$chat-bubble-padding: null !default;
+$chat-bubble-line-height: null !default;
+$chat-bubble-container-gap: 4px !default;
+$chat-messagebox-padding: null !default;
+$chat-messagebox-textarea-min-height: null !default;
+$chat-messagebox-textarea-border-radius: null !default;
+$chat-messagebox-button-height: null !default;
+$chat-messagebox-button-width: null !default;
+$chat-messagebox-gap: 8px !default;
+$chat-messagelist-padding: null !default;
+$chat-messagegroup-gap: 4px !default;
+$chat-messagegroup-padding-top: null !default;
+$chat-messagegroup-author-name-gap: 8px !default;
+$chat-avatar-size: 32px !default;
+$chat-avatar-gap: 12px !default;
+$chat-messagegroup-alignment-start-template: $chat-avatar-size + $chat-avatar-gap 1fr !default;
```

Comment:
Use 'auto 1fr', for space between avatar and the second column use 'column-gap'

Reply:
[b5bbab3e3d2663f50b15f1c50726ddb07f6b6094]